File: exchange-desktop-hmekhatib-main/src/main/java/com/hmekhatib/exchange/api/model/Stat.java
package com.hmekhatib.exchange.api.model;

import com.google.gson.annotations.SerializedName;

public class Stat {
    @SerializedName("1h")
    public Float OneHChange;
    @SerializedName("12h")
    public Float TwelveHChange;
    @SerializedName("24h")
    public Float TwentyFourHChange;
    @SerializedName("7d")
    public Float SevenDChange;
    @SerializedName("max")
    public Float maxRate;
    @SerializedName("min")
    public Float minRate;

    public Float getOneHChange() {
        return OneHChange;
    }

    public Float getTwelveHChange() {
        return TwelveHChange;
    }

    public Float getTwentyFourHChange() {
        return TwentyFourHChange;
    }

    public Float getSevenDChange() {
        return SevenDChange;
    }

    public Float getMaxRate() {
        return maxRate;
    }

    public Float getMinRate() {
        return minRate;
    }
}

File: exchange-android/exchange-android-hmekhatib/app/src/main/java/com/hmekhatib/currencyexchange/api/model/Stat.kt
package com.hmekhatib.currencyexchange.api.model

import com.google.gson.annotations.SerializedName
import java.util.*

class Stat {
    @SerializedName("1h")
    var OneHChange: Float? = null
    @SerializedName("12h")
    var TwelveHChange: Float? = null
    @SerializedName("24h")
    var TwentyFourHChange: Float? = null
    @SerializedName("7d")
    var SevenDChange: Float? = null
    @SerializedName("max")
    var maxRate: Float? = null
    @SerializedName("min")
    var minRate: Float? = null
}